<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsGeneral</class>
 <widget class="QWidget" name="SettingsGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>423</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupTano">
     <property name="title">
      <string>Tano</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="labelVersion">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkWizard">
        <property name="text">
         <string>Run First run wizard after restart</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="spacerTano">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="QLabel" name="labelSettingsInfo">
        <property name="text">
         <string>&lt;i&gt;Note: These settings are shared with all Tano applications.&lt;/i&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelLogoPlayer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../data/logo.qrc">:/logo/64x64/logo.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLogoEditor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../data/logo.qrc">:/logo/64x64/logo-editor.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <layout class="QVBoxLayout" name="layoutLangBox">
      <item>
       <widget class="QRadioButton" name="radioDefault">
        <property name="text">
         <string>System default language</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioCustom">
        <property name="text">
         <string>Custom language</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboLanguage">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDefaultPlaylist">
     <property name="title">
      <string>Default channel list</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="BrowseWidget" name="playlist" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSession">
     <property name="title">
      <string>Session store</string>
     </property>
     <layout class="QVBoxLayout" name="layoutSessionBox">
      <item>
       <widget class="QCheckBox" name="checkVolume">
        <property name="text">
         <string>Remember volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkAutoplay">
        <property name="text">
         <string>Autoplay last channel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowseWidget</class>
   <extends>QWidget</extends>
   <header>ui/core/BrowseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/logo.qrc"/>
 </resources>
 <connections/>
</ui>
